Bonsai.Ximea/Utilities.cs

```diff
@@ -0,0 +1,109 @@
+﻿using OpenCV.Net;
+using System;
+using xiApi.NET;
+
+namespace Bonsai.Ximea
+{
+
+    public enum PixelFormat
+    {
+        //     8 bits per pixel. [Intensity] (see Note5,Note6)
```

Comment:
```suggestion
        /// <summary>
        /// 8 bits per pixel. [Intensity] (see Note5,Note6)
        /// </summary>
```

If we turn these comments into XML docs we could at a later date generate doc pages for the package more easily.

Reply:
Done!